styles/modules/_pageNav.scss

```diff
@@ -148,8 +148,7 @@
 
   .navListWrapper {
     z-index: 1;
-    padding: 0 $pad $pad $pad; // give room for border and shadow
-    right: $pad; // compensate for padding
+    padding: 0 0 $pad $pad; // give room for border and shadow
```

Comment:
This removes the shadow on the right side, which is probably not what you intended. 

Instead, leave the padding as-is, and change the right to negative, which is what it should have been.